documentation/modules/exploit/windows/local/ntusermndragover.md

```diff
@@ -1,8 +1,8 @@
 ## Vulnerable Application
 
-This module exploits a null pointer deference in win32k triggered by the NTUserMNDragOver syscall.
-After receiving the menu window object returned by the window procedure function, the xxxMNFindWindowFromPoint function does not effectively check the validity of its member tagPOPUPMENU, causing the subsequent MNGetpItemFromIndex function to trigger the NULL pointer deference.
-This module has been tested against Windows 7 x86 SP0 and SP1. Offsets within the solution may need to be adjusted to work with other versions of Windows.
+This module exploits a NULL pointer dereference vulnerability in the win32 NtUserMNDragOver() syscall.
+The NULL pointer dereference occurs because the xxxMNFindWindowFromPoint() function does not effectively check the validity of the tagPOPUPMENU objects it processes before passing them on to MNGetpItemFromIndex(), where the NULL pointer dereference will occur.
```

Comment:
Following two paragraphs look good, thanks for updating this!

Reply:
Woops this was meant to be the two paragraphs below these. I don't know why GitHub highlighted these two paragraphs as the comment above before this one is meant to apply to them...sorry about that @timwr.